src/components/Button/_button.scss

```diff
@@ -1,16 +1,36 @@
 .btn {
   @extend %btn-base;
-  background: var(--colour-blue-60);
   border-radius: calc(0.25 * var(--space-unit));
-  color: var(--colour-white);
   display: inline-flex;
   font-family: var(--font-primary);
+  font-size: var(--text-base-size);
+  transition: ease 0.4s;
+  transition-property: background, border, color;
+}
+
+.btn[disabled] {
+  color: var(--colour-grey-50);
+  cursor: not-allowed;
+
+  @include hocus {
+    color: var(--colour-grey-50);
+  }
+}
+
+.btn--primary,
+.btn--secondary,
+.btn--ghost {
   height: calc(5.5 * var(--space-unit));
   justify-content: center;
   min-height: calc(5.5 * var(--space-unit));
   min-width: calc(5.5 * var(--space-unit));
-  padding: 0 calc(4 * var(--space-unit));
-  transition: background ease 0.4s;
+  padding: 0 calc(2.5 * var(--space-unit));
+  text-align: center;
+}
+
+.btn--primary {
+  background: var(--colour-blue-60);
```

Comment:
out of interest - any rationale behind using `background` and not `background-color`?

Reply:
less to type is all

Reply:
Read this recently and made me think we shouldn't use shorthand https://csswizardry.com/2016/12/css-shorthand-syntax-considered-an-anti-pattern/

(exceptions being margin & padding)

Reply:
fair point, updated